.travis.yml

```diff
@@ -62,6 +62,9 @@ script:
     else
       set -e
       bash .misc/tests.sh
+      if [ $? -eq 0 ]; then
```

Comment:
No. All checks. The 'script' section must finish.
See https://docs.travis-ci.com/user/deployment